src/components/AuthPage/Login/index.js

```diff
@@ -104,119 +174,166 @@ const Login = () => {
-            label="Email"
-            variant="outlined"
-            placeholder="mail@codelabz.com"
-            value={email}
-            onChange={onChangeEmail}
-            helperText={emailValidateError ? emailValidateErrorMessage : null}
-            error={emailValidateError}
-            fullWidth
-            autoComplete="email"
-            required
-            onFocus={onFocusEmail}
-            style={{ marginBottom: "15px" }}
-            InputProps={{
-              startAdornment: (
-                <InputAdornment position="start">
-                  <MailOutlined style={{ color: "rgba(0,0,0,.25)" }} />
-                </InputAdornment>
-              ),
-            }}
+    <Grid container className={classes.root}>
+      <Grid
+        style={{
+          display: "flex",
+          justifyContent: "flex-start",
+          alignContent: "left",
+        }}
+        className={`${classes.rootChildrenLeft}`}
+      >
+        <Grid className={classes.loginLeft}>
+          <img
+            src={LoginImg}
+            style={{ height: "auto", width: "auto", marginTop: "5rem" }}
+            alt="login"
           />
-          <TextField
-            label="Password"
-            variant="outlined"
-            helperText={
-              passwordValidateError ? passwordValidateErrorMessage : null
-            }
-            error={passwordValidateError}
-            fullWidth
-            required
-            value={password}
-            onFocus={onFocusPassword}
-            onChange={onChangePassword}
-            autoComplete="current-password"
-            type={showPassword ? "text" : "password"}
-            style={{ marginBottom: "15px" }}
-            InputProps={{
-              startAdornment: (
-                <InputAdornment position="start">
-                  <LockOutlined style={{ color: "rgba(0,0,0,.25)" }} />
-                </InputAdornment>
-              ),
-              endAdornment: (
-                <InputAdornment position="end">
-                  <IconButton
-                    aria-label="toggle password visibility"
-                    onClick={handleClickShowPassword}
-                    onMouseDown={handleMouseDownPassword}
-                  >
-                    {showPassword ? <Visibility /> : <VisibilityOff />}
-                  </IconButton>
-                </InputAdornment>
-              ),
-            }}
-          />
-          <Grid container alignItems="center" justify="space-between">
-            <Grid>
-              <FormGroup row>
-                <FormControlLabel
-                  control={
-                    <Checkbox
-                      // checked={state.checkedB}
-                      // onChange={handleChange}
-                      name="remember"
-                      color="primary"
+        </Grid>
+
+        <Card
+          raised
+          className={`${classes.card} ${classes.loginRight}  `}
+          // className=
```

Comment:
actually there is lots of current because I created different class for different looks , thinking if current design is not approved I'll change with different class. Now current design is approved so I'll remove all unnecessary code and I'll commit them